post-build/action.yml

```
@@ -78,14 +68,25 @@ runs:
         name: bottles-${{ inputs.runner }}
         path: ${{ inputs.bottles-directory }}/failed
 
+    - name: Upload list of successfully tested dependents
+      if: always() && !fromJson(inputs.upload-bottles) && steps.bottles.outputs.dependents > 0
+      uses: actions/upload-artifact@main
+      with:
+        name: dependents
+        path: ${{ inputs.bottles-directory }}/tested-dependents-*.txt
+
     # Must be run before the `Upload bottles` step so that failed
     # bottles are not included in the `bottles` artifact.
-    - name: Delete failed bottles
+    - name: Delete unneeded files
       if: always()
-      run: rm -rvf "${BOTTLES_DIR:?}/failed"
+      run: |
+        rm -rvf logs
+        rm -rvf home
+        rm -rvf failed
+        rm -rvf artifact-cache
+        rm -rvf successful-dependents-*.txt
```

```suggestion
        rm -rvf logs home failed artifact-cache
        rm -rvf successful-dependents-*.txt
```

As I think everything except the glob will succeed regardless.

Longer-term: makes me think that it'd be nice to put all of these in a single `test_bot` root directory or something.